### ISSUE TITLE
Trading chart shows no candles if you use 1m Interval 

### ISSUE THREAD

**Issue by bwallacee** [vegaprotocol/frontend-monorepo#5469]
## Description

The trading chart isn't showing any candles if you select 1m interval. This only seems to appear if there hasn't been any orders in a while. e.g. on a closed market or a market that has been suspended.

## Steps to Reproduce

1. Go to https://vegafairground.eth.limo/#/markets/fd39bec848535f4c1e6842cb33f6d626ae3e905b6260aef00a0c48b82d237a20
2. Click on 1m interval
3. See error

## Expected behaviour

Trading chart shows candles no matter the interval 

## Screenshots

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/133853298/06aced1a-45cd-47d0-b301-fd5b79bd501d)
![image](https://github.com/vegaprotocol/frontend-monorepo/assets/133853298/3540a5c8-2534-4a70-968c-2d3c263cfad9)

**Comment by mattrussell36** [comment]
Candles are null, in this case the market was suspended by governance

![Image](https://github.com/vegaprotocol/frontend-monorepo/assets/6803987/be9b35ac-8ff8-4224-8df5-0bfd6275f4e2)

**Comment by mattrussell36** [comment]
Possibly related to https://github.com/vegaprotocol/vega/issues/10219

**Comment by bglownia** [comment]
If there is no data for requested date range empty array is returned by VegaDataSource. After that one which was returned through subscription is added. In effect we got chart with single candle
![image](https://github.com/vegaprotocol/frontend-monorepo/assets/1754247/9b440d70-763c-4a3a-b6ad-b491d0a90174)
After user will try to zoom this candle is used to calculate new date range and user will get more candles (but they will from the past).
To avoid it I added condition that checks if candle from subscription is newer than beginning of requested persion
